rdt-open-reader/src/test/java/io/ona/rdt/robolectric/shadow/UtilsShadow.java

```diff
@@ -41,4 +44,12 @@ public static synchronized void setMockCounter(MockCounter mockCounter) {
     public static synchronized MockCounter getMockCounter() {
         return mockCounter;
     }
+
+    public static void verifyUserAuthorization(Context context) {
+        UtilsShadow.context = context;
+    }
+
+    public static Context getActiveContext() {
+        return context;
+    }
```

Comment:
Remove this and use the `mockCounter` instead to verify calls to methods.

Reply:
fixed.